source/_docs/redirects.md

```diff
@@ -134,6 +134,42 @@ if (isset($_ENV['PANTHEON_ENVIRONMENT']) && ($_ENV['PANTHEON_ENVIRONMENT'] === '
 }
 ```
 
+### Redirect Using Wildcards
+The following configuration will redirect requests for `example.com` using wildcard regex patterns:
+
+```php
+// Add wildcard redirects (old_path(s) => new_path)
+$redirects = array(
+  "/any/old/path/(.*)" => "/new",
+  "/starts-with(.*)" => "/new/path",
+  "/(.*)ends-with-this" => "/new/path",
+  "/(.*)contains-this(.*)" => "/new/path",
+  "/node/(\d)+" => "/all-nodes-go-here",
+  "/user/(a-z)+/blog" => "/new-user-blogs",
+);
+
+$match = FALSE;
+foreach ($redirects as $old_path => $new_path) {
+  if ($match = preg_match("#^($old_path)$#", $_SERVER['REQUEST_URI'])) {
```

Comment:
This should be optimized to a single preg_match without a `foreach`, e.g.:
```
if (preg_match('#^(/old/path|/other/path)$', $_SERVER['REQUEST_URI']))) {
```

Reply:
This will work if you wanted all the old paths to redirect to a single new path but what if you wanted each old path to redirect to a unique new path?

Reply:
I imagine the current code could get bogged down if you had a lot of redirects but I can't imagine it would make much of a difference in performance with just a handful of redirects.